CRM/Core/BAO/CustomField.php

```diff
@@ -555,7 +555,7 @@ public static function &getFields(
 
         $fields = [];
         while (($dao->fetch()) != NULL) {
-          $regexp = preg_replace('/[.,;:!?]/', '', NULL);
+          $regexp = preg_replace('/[.,;:!?]/', '', '');
```

Comment:
What does this line do? Is it here to create a nanosecond delay?

Reply:
🤷

Reply:
@eileenmcnaughton Any thoughts on this part? Was added at https://github.com/civicrm/civicrm-core/commit/cef2e96c882537457f8559e3d3d351e24eb2dad2